### ISSUE TITLE
tf.nn.softmax_cross_entropy_with_logits Erro

### ISSUE THREAD

**Issue by andreseloysv** [llSourcell/tensorflow_neural_net_live_demo#1]
The solution is:

cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits =pred, labels=y))
